biz.aQute.repository/src/aQute/maven/nexus/provider/Nexus.java

```diff
@@ -122,13 +122,16 @@ private boolean isReal(URI uri) {
 	public File download(URI uri) throws Exception {
 		return request().useCache()
 			.age(30, TimeUnit.SECONDS)
+			.timeout(5 * 60 * 1000)
+			.retries(3)
 			.go(uri);
 	}
 
 	public void upload(URI uri, byte[] data) throws Exception {
 		try (TaggedData tag = request().put()
 			.upload(data)
 			.asTag()
+			.retries(3)
```

Comment:
Currently retries are limited to `GET`. So we would need to change HttpClient to support them for other http verbs.

https://github.com/bndtools/bnd/blob/6ca9a2214a422eaa93fae07ec34aa6624885d385/biz.aQute.bndlib/src/aQute/bnd/http/HttpClient.java#L153